### PR TITLE
LUN-125: Articles in overview misplaced on bigger screens

### PR DESCRIPTION
**Pull request by ansamshtaiwi1**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/routes/vocabulary-list/components/VocabularyListItem.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const StyledTitle = styled.Text`
   overflow: hidden;
   height: ${wp('5%')};
   text-align: center;
-  line-height: 18;
+
```
Comment (klinzo) [inline src/routes/vocabulary-list/components/VocabularyListItem.tsx:52]:
```suggestion
```

**Review by klinzo — APPROVED** [review]
Tested on Android, fixes the issue! Well done :)

**Review by Taggotty — APPROVED** [review]
Nice :)
Is there a reason why you chose LUN-13 as base branch? It's hard to find the relevant change under all those files.

**Comment by YasminNajar** [comment]
@Taggotty I thought it was better to make the changes directly to the new styleComponents but now I believe it's better to do it on the main branch instead as it's a small change that can be easily applied. So I made a new branch from main and I will push it and close this one if that's okay.

**Comment by Taggotty** [comment]
> @Taggotty I thought it was better to make the changes directly to the new styleComponents but now I believe it's better to do it on the main branch instead as it's a small change that can be easily applied. So I made a new branch from main and I will push it and close this one if that's okay.

Sounds like a great idea actually 🙈

**Comment by klinzo** [comment]
@YasminNajar can we delete this branch or do you still need it?

**Pull request closed without merging**